botfront/private/defaults/private.dev.yaml

```diff
@@ -21,16 +21,7 @@ policies:
     max_history: 5
     batch_size: [32, 64]
   - name: RulePolicy
-    enable_fallback_prediction: false
   - name: AugmentedMemoizationPolicy
```

Comment:
Shouldn't `AugmentedMemoizationPolicy` be removed ? Isn't it covered by `RulePolicy`

Reply:
`AugmentedMemoizationPolicy` is not deprecated. It handles stories, like `TEDPolicy`.